addons/project/models/project_task.py

```diff
@@ -2011,3 +2012,25 @@ def _get_thread_with_access(self, thread_id, mode="read", **kwargs):
             ):
                 kwargs["token"] = token
         return super()._get_thread_with_access(thread_id, mode, **kwargs)
+
+    def get_mention_suggestions(self, search, limit=8):
+        """Return the 'limit'-first followers of the given task or followers of its project matching
+        a 'search' string as a list of partner data (returned by `_to_store()`).
+        See similar method for all partners `get_mention_suggestions()`.
+        """
+        self.ensure_one()
+        project = self.project_id
+        if not (
+            project
+            and project._check_project_sharing_access()
+            and project._get_thread_with_access(project.id)
+        ):
+            return {}
+        # sudo: mail.followers - reading message_follower_ids on accessible task/project is allowed
+        followers = project.sudo().message_follower_ids | self.sudo().message_follower_ids
```

Comment:
Probably ok for now, but... If the list of followers is very long, this will need to fetch the whole table.
Better if possible build the follower list into the main query as a sub query or join.